### PR TITLE
Changed wrong sample on view-models.md

### PR DESCRIPTION
**Pull request by Vitancourt**
## Purpose of this pull request

This pull request (PR) will fix wrong sample code on view-models.md

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.2/extension-dev-guide/view-models.html
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/view-models.html

-  ...

## Links to Magento source code

https://github.com/magento/devdocs/blob/master/guides/v2.2/extension-dev-guide/view-models.md

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@Vitancourt thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/extension-dev-guide/view-models.md [hunk 1]
```diff
@@ -39,23 +39,18 @@ class MyClass implements \Magento\Framework\View\Element\Block\ArgumentInterface
...
 ```
 
 Finally, in the `cart/item/default.phtml` template, you can access the public methods of your view model:
 
-```html
-<?php
-
-/** @var $viewModel \Vendor\CustomModule\ViewModel\MyClass */
-
+```php
```
Comment (nuzil) [inline guides/v2.2/extension-dev-guide/view-models.md]: I would not say its correct. Yes we have to highlight code, but its a phtml file, and <h1> usage was fine here

Comment (Vitancourt) [reply guides/v2.2/extension-dev-guide/view-models.md]: @nuzil fixed, i just keep the php part

**Comment by dobooth** [comment]
Closing this. I JUST merged another PR that fixed this same error. Appreciate the work, though.

**Pull request closed without merging**